[2025-07-28T19:30:32.702+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-28T19:30:32.783+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: financial_etl_pipeline.load_data manual__2025-07-28T19:17:11+00:00 [queued]>
[2025-07-28T19:30:32.826+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: financial_etl_pipeline.load_data manual__2025-07-28T19:17:11+00:00 [queued]>
[2025-07-28T19:30:32.828+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2025-07-28T19:30:33.236+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_data> on 2025-07-28 19:17:11+00:00
[2025-07-28T19:30:33.253+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'financial_etl_pipeline', 'load_data', 'manual__2025-07-28T19:17:11+00:00', '--job-id', '176', '--raw', '--subdir', 'DAGS_FOLDER/financial_etl_dag.py', '--cfg-path', '/tmp/tmpl86tz4zb']
[2025-07-28T19:30:33.260+0000] {standard_task_runner.py:91} INFO - Job 176: Subtask load_data
[2025-07-28T19:30:33.260+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2319) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-28T19:30:33.262+0000] {standard_task_runner.py:63} INFO - Started process 2322 to run task
[2025-07-28T19:30:33.392+0000] {task_command.py:426} INFO - Running <TaskInstance: financial_etl_pipeline.load_data manual__2025-07-28T19:17:11+00:00 [running]> on host 7815b21b8845
[2025-07-28T19:30:33.584+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='financial_etl_pipeline' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-28T19:17:11+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-28T19:17:11+00:00'
[2025-07-28T19:30:33.594+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-28T19:30:33.972+0000] {logging_mixin.py:188} INFO - Data loaded into PostgreSQL 'transactions' table.
[2025-07-28T19:30:33.973+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-28T19:30:33.977+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-28T19:30:34.030+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=financial_etl_pipeline, task_id=load_data, execution_date=20250728T191711, start_date=20250728T193032, end_date=20250728T193034
[2025-07-28T19:30:34.082+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-28T19:30:34.140+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-28T19:30:34.144+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
